### DISCUSSION TITLE
初次运行报错

### DISCUSSION THREAD

**Discussion by zq12580** [ant-design/ant-design-mobile#4007] [Q&A] [answered]
These dependencies were not found:
* antd-mobile/es/button in ./src/pages/index.jsx
* antd-mobile/es/button/style in ./src/pages/index.jsx
To install them, you can run: npm install --save antd-mobile/es/button antd-mobile/es/button/style
怎么解决？？？

**Comment by awmleer** [comment, marked as answer]
可以参考下这里：https://github.com/ant-design/ant-design-mobile/issues/3962#issuecomment-918857140

**Reply by zq12580** [reply]
谢谢，已解决
